SYNTAX.md

```diff
@@ -44,5 +44,6 @@ All those three items can be combined with `name` property to be show in logs.
 - `method`: HTTP method in the requests. Valid methods are GET, POST, PUT, PATCH or DELETE. (default: GET)
 - `body`: Request body for methods like POST, PUT or PATCH.
 - `with_items`: List of items to be interpolated in the given request url.
+- `with_items_iter`: Generates items from an iterator from start, step, stop.
```

Comment:
What about changing the name to `with_items_range`? I think it fits better with the functionality.

Reply:
done